samza-core/src/main/java/org/apache/samza/config/JobConfig.java

```diff
@@ -341,4 +344,13 @@ public static String getFwkPath(Config config) {
           new File(dir, String.format(CONTAINER_METADATA_FILENAME_FORMAT, execEnvContainerId)));
     }
   }
+
+  /**
+   * Get coordinatorStreamFactory according to the configs
+   * @return the name of coordinatorStreamFactory
+   */
+  public static String getCoordinatorStreamFactory(Config config) {
```

Comment:
The pattern for the config classes is to make this an instance method, instead of a static method.